plugin-server/src/main/ingestion-queues/batch-processing/each-batch-ingestion.ts

```diff
@@ -22,6 +24,25 @@ export async function eachMessageIngestion(message: KafkaMessage, queue: Ingesti
         return
     }
 
+    if (
+        defaultConfig.KAFKA_CONSUMPTION_TOPIC != KAFKA_EVENTS_PLUGIN_INGESTION_OVERFLOW &&
```

Comment:
Hmm I think we'll need to update this conditional to check that the message is not from the overflow topic 🤔 

I've got a feeling we don't have that info at this point, and that we'll need to go up to the batch level to get this.